### DISCUSSION TITLE
Default task or operator from failed task

### DISCUSSION THREAD

**Discussion by RuslanYaku** [apache/airflow#25379] [Ideas]
Hi all!
There is an idea to create the possibility of redirecting tasks within a dag.
For example, there is
task1
task2
task3 etc
start >> [ task1, task2, task3 ] >> [ task4, task5, task6 ] >> end

and if task4 ended with failed status, then automatically redirect it to FailedTasksOperator, for example. This will give the option to restart task4 so that it does not restart all downstream tasks.

If such a functionality already exists, and I just don’t know it, then please direct me))

![image](https://user-images.githubusercontent.com/79851076/181580711-4420c528-3f0f-4243-b0a3-55aeca9f02d1.png)

you can see in the picture that after failed nothing goes if no retry parameter is specified. 
Maybe I'm suggesting some weird topic, but I would like to hear your opinion.

**Comment by potiuk** [comment]
You can program it in a similar way to:

```
for task in dag.tasks:
   # add "on failure" dependency on your task
```

This is Python you can add loops and imperative programming in the DAG.

**Reply by RuslanYaku** [reply]
And it will redirect task operator flow at runtime in another operator "FailedOperator"?
Or do you mean to code the actions of the PythonOperator if it catches an error?
